security/cmd/istio_ca/main.go

```diff
@@ -61,6 +61,11 @@ const (
 
 	// The key for the environment variable that specifies the namespace.
 	listenedNamespaceKey = "NAMESPACE"
+
+	// ServiceAccount/DNS pair for generating DNS names in certificates.
+	// TODO: move it to a configmap later when we have more services to support.
```

Comment:
Pilot and Mixer validation webhooks need certs provisioned so I can add them to default Istio install.

Reply:
Do they also have separate service accounts like sidecar injector?

Reply:
Yes, `istio-pilot-service-account` and `istio-mixer-validator-service-account`.

Reply:
Shouldn't we move the pilot validator service out of pilot service?

Reply:
Yes, eventually both pilot and mixer validation webhooks could be merged into a single config validation service separate from Mixer and Pilot proper.